### ISSUE TITLE
大神！想请教下mtcnn_euclidean_loss_layer.cpp代码问题

### ISSUE THREAD

**Issue by lbhnrg2021** [dlunion/CCDL#20]
在53行处，根据欧式距离公式，应该还要除以一个处理过的数据项数量，但是我发现您没有做这步计算，是不是我对公式理解有误？

**Comment by lbhnrg2021** [comment]
我做了这么修改
53-line:loss += dot;count++;
57-line:top[0]->mutable_cpu_data()[0] = loss/count/2;
12Pnet的loss一直在0.02-0.04徘徊，也不知道对不对

**Comment by Aliang-SEU** [comment]
应该是要做norm的

**Comment by lbhnrg2021** [comment]
谢谢大神指点：）
